examples/web-color/README.md

```diff
@@ -0,0 +1,146 @@
+# Web color example
+
+This example shows the mechanics of applying patches to a register. The data
+is the list of web colors as defined by the [W3C](https://www.w3c.org), in
+particular [HTML4](https://www.w3.org/TR/1999/REC-html401-19991224/) and
+[CSS Color Module Level 3](https://www.w3.org/TR/css-color-3/).
+
+
+## Setup
+
+Ensure that you have Python +3.6 installed on your machine.
+
+Then, install the `registers` command line interface:
+
+```sh
+pip3 install --user "git+https://github.com/openregister/registers-cli#egg=registers"
+```
+
+Then verify it works:
+
+```sh
+$ registers --help
+Usage: registers [OPTIONS] COMMAND [ARGS]...
+
+  registers lets you manage a Register represented as a Register
+  Serialisation Format (RSF).
+```
+
+### Troubleshooting
+
+* **The 'registers' distribution was not found**: This is likely to be
+  Pip configuration issue. Read more here:
+  https://pip.pypa.io/en/stable/user_guide/#user-installs
+* **There are conflicting dependencies**: Python manages dependencies
+  centrally. Typically, if you need to isolate them, you would use some sort
+  of virtualenv.
+
+
+## Structure of the example
+
+This example has:
+
+* The **register** for the web colors in `src/web-color.rsf`.
+* The **data** in TSV we will add to the register in `data/`.
+
+Note that the filenames could be anything, the content is what's relevant.
+
+
```

Comment:
Can we state here or at the start of the next section that these commands should be run relative to `examples/web-color`?

e.g. 
> From your checkout directory `cd examples/web-color`?

I know the examples are self contained but from experience people often try to run commands from the checkout directory. e.g. this is how we document them previously in `deployment`.

Reply:
fair point 👍